Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -774,12 +782,14 @@
 		};
 		FF5D13C92A86C2D800BC9BD6 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = FF098EFA2AB3424E003EC0FE /* Basic-Car-Maintenance.xcconfig */;
```

Comment:
This should not be changed. Re-read the README again to properly set up this project